src/vcpkg/binarycaching.cpp

```diff
@@ -2561,9 +2615,8 @@ void vcpkg::help_topic_asset_caching(const VcpkgPaths&)
     tbl.text("The `<rw>` optional parameter for certain strings controls how they will be accessed. It can be "
              "specified as `read`, `write`, or `readwrite` and defaults to `read`.");
     tbl.blank();
-    print2(tbl.m_str);
-
-    print2("\nExtended documentation is available at ", docs::assetcaching_url, "\n");
+    msg::println(msgHelpTableFormat, msg::value = tbl.m_str);
```

Comment:
Here the content that actually needs to be localized is the help text blocks above rather than this part.

Reply:
Same as before